Fix IVF-PQ helper functions (#116)
rapidsai/cuvs: docs/source/cpp_api/neighbors_ivf_pq.rst

    :project: cuvs
    :members:
    :content-only:

> Helper Methods

Reviewer: Can you add a small description for this section? For example, so that a user knows when they might want to use these functions.

    :project: cuvs
    :members:
    :content-only:

Helper Methods
---------------

> Those methods are used to manipulate the underlying data of an IVF-PQ index, and perform operations like unpacking records, or writing PQ codes into an existing IVF list.

Reviewer: ```suggestion
Additional helper functions for manipulating the underlying data of an IVF-PQ index, unpacking records, and writing PQ codes into an existing IVF list.
```
